### ISSUE TITLE
What is weight file size ??

### ISSUE THREAD

**Issue by abhigoku10** [akarshzingade/Logo-Detection-YOLOv2#2]
@akarshzingade thanks for giving an idea for the implementation . wht is the size of the file generated ???
can we use this dataset to check the orientation of an logo

**Comment by akarshzingade** [comment]
Well, the weight file linked in this repository is 275MB. 

I am not sure about the orientation part.

**Comment by abhigoku10** [comment]
@akarshzingade even after using yolo v2 the size of the model is around 275 mb can we minimize the model size ???

After detecting the logo i needed to check the orientation of the logo , ie to say where its horizontal or vertical or at any other angle is it jpossible to identify this 

thanking you in advance

**Comment by akarshzingade** [comment]
Maybe you can use different architectures for YOLOv2- like tinyYOLO. It has a lower memory footprint. 

Once you get the bounding box for your logo, you can use orientation detection techniques to find the orientation. A quick glance google showed this: https://github.com/akarshzingade/Logo-Detection-YOLOv2/issues/2#issuecomment-366869826. I hope it gives you a good starter.

**Comment by ghost** [comment]
help me in the instruction of obj.names 
i didnot understand this instruction,!!
3.Path to store the train.txt/test.txt file and obj.names file.
4.Name of the text file to store the paths to the images for train/test.

**Comment by abhigoku10** [comment]
@mathan1312  these are the same steps in the yolov2 training where the test and train files in the txt format has all the list of file which has to be trained

**Comment by ghost** [comment]
yeah @abhigoku10 , I think i given the path for input folder , then output folder . thereafter it created some of images and .text files. i don't know what to do further for testing my own images

**Comment by m4x1mus** [comment]
How do I chnage the predicted labels if I am using the pretrained logo detection model?
On running 
`./darknet detect cfg/yolov2_logo_detection.cfg YOLOv2_logo_detection_10000th_iteration.weights data/logo.jpg ` 
for the image attached I get
`data/logo.jpg: Predicted in 7.145362 seconds.
wine glass: 77%` 
which is not the correct label (and not present in label set in obj.names)
![logo](https://user-images.githubusercontent.com/43358553/61414054-eafd7600-a8e4-11e9-8538-387dcb2cda95.jpg)
